### PR TITLE
Merge `main` into `feature/review-workflow-multiple` (including bulk publish)

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Merges `main` into `feature/review-workflow-multiple` (including bulk publish).

### Why is it needed?

Keep up to date with main.

### Related issue(s)/PR(s)

- Builds on top of https://github.com/strapi/strapi/pull/17336 and therefore
- Closes https://github.com/strapi/strapi/pull/17336
- Refs https://github.com/strapi/strapi/pull/16755

### CODE REVIEW

File: packages/utils/babel-plugin-switch-ee-ce/package.json
Comment (joshuaellis) [inline packages/utils/babel-plugin-switch-ee-ce/package.json:1]: this shouldn't be here?

Comment (gu-stav) [reply packages/utils/babel-plugin-switch-ee-ce/package.json:1]: That is very true. Why is that? I'll investigate.

Comment (gu-stav) [reply packages/utils/babel-plugin-switch-ee-ce/package.json:1]: For some reason this file has made it back into `main`: https://github.com/strapi/strapi/tree/main/packages/utils/babel-plugin-switch-ee-ce

Comment (joshuaellis) [reply packages/utils/babel-plugin-switch-ee-ce/package.json:1]: for fucks sake

**Review by joshuaellis — APPROVED** [review]

File: packages/utils/babel-plugin-switch-ee-ce/package.json
Comment (gu-stav) [reply packages/utils/babel-plugin-switch-ee-ce/package.json:1]: Oh I see. @Marc-Roig fyi this is an accident: https://github.com/strapi/strapi/commit/7bdef3d27843c78666c85f30dc2f1bcf4d6b805e#diff-0c24787bb61753b049782d9abba3055411f9d2c558941fa96712bd74239e297b

I'll remove the file from `main` again.

Comment (gu-stav) [reply packages/utils/babel-plugin-switch-ee-ce/package.json:1]: https://github.com/strapi/strapi/pull/17353

**Pull request merged**